### DISCUSSION TITLE
总体是refresh_token无效

### DISCUSSION THREAD

**Discussion by rocky-peng** [messense/aliyundrive-webdav#799] [Q&A]
### 问题描述

用docker部署，总提示
```
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
```

### 重现步骤

1. centos
2. docker
```
docker run -d --name=aliyundrive-webdav --restart=unless-stopped -p 18080:8080 \
  -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
  -e REFRESH_TOKEN='xxxxx' \
  -e WEBDAV_AUTH_USER=admin \
  -e WEBDAV_AUTH_PASSWORD=admin \
  messense/aliyundrive-webdav
```

提示 Error: Invalid refresh token value found in `--refresh-token` argument
我重新登陆阿里云盘获取refresh_token后还是提示同样的错误信息

### 版本

docker怎么看版本哦

### 运行平台

Docker

### 日志

Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument
Error: Invalid refresh token value found in `--refresh-token` argument

**Comment by lhzcc** [comment]
老哥，我也遇到了，你解决了吗

**Comment by loloku798** [comment]
token有最新的获取方式，不再是以前通过浏览器的调试来获取token了。
新的获取方式：[通过在线工具获取 refresh token](https://messense-aliyundrive-webdav-backendrefresh-token-ucs0wn.streamlit.app/)

**Reply by tjbzyl1212** [reply]
谢谢楼上老哥

**Reply by harwinvoid** [reply]
谢谢老哥

**Reply by HWKevin** [reply]
谢谢老哥
